data/tlg1269/tlg002/__cts__.xml

```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg1269" xml:lang="grc" urn="urn:cts:greekLit:tlg1269.tlg002">
+    <ti:title xml:lang="lat">Cleanthis Assii Fragmenta et Apophthegmata.</ti:title>
+    <ti:edition urn="urn:cts:greekLit:tlg1269.tlg002.1st1K-grc1" workUrn="urn:cts:greekLit:tlg1269.tlg002">
+        <ti:label xml:lang="lat">Cleanthis Assii Fragmenta et Apophthegmata.</ti:label>
```

Comment:
Same here. I don't think the book page is the way to go here. If in doubt, check with Alison.
<ti:label xml:lang="lat">Testimonia et fragmenta</ti:label>

No periods in either place.